src/Aeternity/Signer.cpp

```diff
@@ -0,0 +1,80 @@
+// Copyright © 2017-2019 Trust.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include "Signer.h"
+#include "Address.h"
+#include "Base64.h"
+#include "ChecksumEncoder.h"
+#include "HexCoding.h"
+#include "Identifiers.h"
+#include <Data.h>
+#include <Ethereum/RLP.h>
+
+using namespace TW;
+using namespace Aeternity;
+
+/// implementation copied from
+/// https://github.com/aeternity/aepp-sdk-go/blob/07aa8a77e5/aeternity/helpers.go#L367
+Proto::SigningOutput Aeternity::Signer::sign(const TW::PrivateKey &privateKey, Transaction &transaction) {
+    const std::string &txString = transaction.encode();
```

Comment:
can we return `Data`? so we don't need to call `parseRawTransaction`

Reply:
Again, good catch, this method was completely redundant. This is how it's implemented in aeternity sdk, so I just didn't think why it's like this.